scripts/libmetal.mk

```diff
@@ -25,21 +30,21 @@ $(BSP_DIR)/build/Makefile:
 		--with-builtin-libgloss
 	touch -c $@
 
-$(BSP_DIR)/install/stamp: $(BSP_DIR)/build/Makefile
-	$(MAKE) -C $(abspath $(BSP_DIR)/build) install
+$(METAL_LIB_DIR)/stamp: $(BSP_DIR)/build/$(CONFIGURATION)/Makefile
+	$(MAKE) -C $(abspath $(BSP_DIR)/build/$(CONFIGURATION)) install
 	date > $@
 
-$(BSP_DIR)/install/lib/libriscv%.a: $(BSP_DIR)/install/stamp ;@:
+$(METAL_LIB_DIR)/libriscv%.a: $(METAL_LIB_DIR)/stamp ;@:
 
-$(BSP_DIR)/install/lib/libmetal.a: $(BSP_DIR)/install/lib/libriscv__mmachine__$(TARGET).a
+$(METAL_LIB_DIR)/libmetal.a: $(METAL_LIB_DIR)/libriscv__mmachine__$(TARGET).a
 	cp $< $@
 
-$(BSP_DIR)/install/lib/libmetal-gloss.a: $(BSP_DIR)/install/lib/libriscv__menv__metal.a
+$(METAL_LIB_DIR)/libmetal-gloss.a: $(METAL_LIB_DIR)/libriscv__menv__metal.a
 	cp $< $@
 
 .PHONY: clean-metal
 clean-metal:
```

Comment:
The metal-clean target removes all CONFIGURATIONS - is this on purpose?
I would like it only to remove a single CONFIGURATION, just like the clean-software target does.

Reply:
So yeah, right now it's removing the install directory entirely.

For removing a single configuration, it probably makes most sense to remove install/lib/CONFIGURATION. The question then is: when do we remove the installed Metal headers? Do we just leave them, or do we add a check and if we're removing the last config, then also delete the headers?

Reply:
I pushed a commit to do the latter (remove the headers if we're removing the last CONFIGURATION).